astropy/cosmology/_io/tests/test_json.py

```diff
@@ -125,7 +126,7 @@ def test_readwrite_json_subclass_partial_info(
         cosmo.write(fp, format="json")
 
         # partial information
-        with open(fp) as file:
+        with fp.open() as file:
             L = file.readlines()[0]
```

Comment:
If I understand this correctly, you want `readline()` instead of `readlines()[0]`?

Reply:
🤷 if it works

Reply:
good catch. Done !